### PR TITLE
MM-41391 Gekidou - Update Info.plist

### PR DESCRIPTION
**Pull request by avinashlng1080**
#### Summary
This PR updates some keywords in the info.plist file on iOS, in an aim to make them as generic as possible so as to facilitate white-labeling.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-41391

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by enahum** [comment]
I thought workspace is more of a cloud thing.. @cwarnermm I feel we definitely need some clarity regarding this

**Comment by cwarnermm** [comment]
@enahum The term "workspace" has become a common term to represent both Cloud and self-hosted deployments. I'm working with the PM Team to formalize and codify this terminology so we can provide clear guidance on usage for "workspace", "server", and "instance" going forward.

In this specific case, if "workspace" isn't ideal, and brevity is preferred, we could remove the "workspace" references altogether, and simply refer to the product as Mattermost. For example:
- Share contacts within Mattermost.
- Send your current location to Mattermost.
- Capture audio when recording a video to share in Mattermost.

Thoughts?

**Comment by enahum** [comment]
> @enahum The term "workspace" has become a common term to represent both Cloud and self-hosted deployments. I'm working with the PM Team to formalize and codify this terminology so we can provide clear guidance on usage for "workspace", "server", and "instance" going forward.
> 
> In this specific case, if "workspace" isn't ideal, and brevity is preferred, we could remove the "workspace" references altogether, and simply refer to the product as Mattermost. For example:
> 
> * Share contacts within Mattermost.
> * Send your current location to Mattermost.
> * Capture audio when recording a video to share in Mattermost.
> 
> Thoughts?

Yeah I like that better, and we should be using $(PRODUCT_NAME) instead. as per iOS documentation
> PRODUCT_NAME The app name of the product built by the current scheme

That way for those who white label the app their name is used right away in the permissions dialogs

**Comment by cwarnermm** [comment]
@enahum - Perfect! Love the placeholder usage approach too. Since we're aligned on simplifying the language overall to avoid the use of the term "workspace", I'll provide proposals for the string updates directly in this PR.

File: ios/Mattermost/Info.plist [hunk 1]
```diff
@@ -64,25 +64,25 @@
 	<key>NSBluetoothPeripheralUsageDescription</key>
 	<string>Share post data across devices with Mattermost</string>
 	<key>NSCalendarsUsageDescription</key>
-	<string>Share contacts in your Mattermost instance</string>
+	<string>Share contacts in your Mattermost workspace</string>
```
Comment (cwarnermm) [inline ios/Mattermost/Info.plist:67]:
```suggestion
	<string>Share contacts within Mattermost</string>
```

File: ios/Mattermost/Info.plist [hunk 2]
```diff
@@ -64,25 +64,25 @@
...
 	<key>NSCameraUsageDescription</key>
-	<string>Take a Photo or Video and upload it to your Mattermost instance</string>
+	<string>Take a Photo or Video and upload it to your Mattermost workspace</string>
```
Comment (cwarnermm) [inline ios/Mattermost/Info.plist:69]:
```suggestion
	<string>Take a Photo or Video and upload it to Mattermost</string>
```

File: ios/Mattermost/Info.plist [hunk 3]
```diff
@@ -64,25 +64,25 @@
...
 	<key>NSFaceIDUsageDescription</key>
 	<string>Mattermost uses Face ID to restrict unauthorized users from accessing the app.</string>
 	<key>NSLocationAlwaysUsageDescription</key>
-	<string>Send your current location to your Mattermost instance</string>
+	<string>Send your current location to your Mattermost workspace</string>
```
Comment (cwarnermm) [inline ios/Mattermost/Info.plist:73]:
```suggestion
	<string>Send your current location to Mattermost</string>
```

Comment (cwarnermm) [reply ios/Mattermost/Info.plist:73]: Question: Does a mobile user **send** their current location to an app, or do they **share** their current location with an app?

File: ios/Mattermost/Info.plist [hunk 4]
```diff
@@ -64,25 +64,25 @@
...
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>Send your current location to your Mattermost instance</string>
+	<string>Send your current location to your Mattermost workspace</string>
```
Comment (cwarnermm) [inline ios/Mattermost/Info.plist:75]:
```suggestion
	<string>Send your current location to Mattermost</string>
```

Comment (cwarnermm) [reply ios/Mattermost/Info.plist:75]: Same question as above. Also - why are there 2 strings that appear to be communicating the same message?

File: ios/Mattermost/Info.plist [hunk 5]
```diff
@@ -64,25 +64,25 @@
...
 	<key>NSMicrophoneUsageDescription</key>
-	<string>Capture audio when recording a video to share to your Mattermost instance</string>
+	<string>Capture audio when recording a video to share to your Mattermost workspace</string>
```
Comment (cwarnermm) [inline ios/Mattermost/Info.plist:77]:
```suggestion
	<string>Capture audio when recording a video to share within Mattermost</string>
```

File: ios/Mattermost/Info.plist [hunk 6]
```diff
@@ -64,25 +64,25 @@
...
 	<key>NSMotionUsageDescription</key>
-	<string>Share your route in your Mattermost instance</string>
+	<string>Share your route in your Mattermost workspace</string>
```
Comment (cwarnermm) [inline ios/Mattermost/Info.plist:79]:
```suggestion
	<string>Share your route in Mattermost</string>
```

Comment (cwarnermm) [reply ios/Mattermost/Info.plist:79]: Given that the user **shares** a route in an app, perhaps users do **share** their location information with an app too?

File: ios/Mattermost/Info.plist [hunk 7]
```diff
@@ -64,25 +64,25 @@
...
 	<key>NSPhotoLibraryAddUsageDescription</key>
-	<string>Save downloaded Photos and Videos from your Mattermost instance to your device</string>
+	<string>Save downloaded Photos and Videos from your Mattermost workspace to your device</string>
```
Comment (cwarnermm) [inline ios/Mattermost/Info.plist:81]:
```suggestion
	<string>Save downloaded Photos and Videos from Mattermost to your device</string>
```

File: ios/Mattermost/Info.plist [hunk 8]
```diff
@@ -64,25 +64,25 @@
...
 	<key>NSPhotoLibraryUsageDescription</key>
-	<string>Upload Photos and Videos from your device to your Mattermost instance</string>
+	<string>Upload Photos and Videos from your device to your Mattermost workspace</string>
```
Comment (cwarnermm) [inline ios/Mattermost/Info.plist:83]:
```suggestion
	<string>Upload Photos and Videos from your device to Mattermost</string>
```

File: ios/Mattermost/Info.plist [hunk 9]
```diff
@@ -64,25 +64,25 @@
...
 	<key>NSSpeechRecognitionUsageDescription</key>
-	<string>Send voice messages to your Mattermost instance</string>
+	<string>Send voice messages to your Mattermost workspace</string>
```
Comment (cwarnermm) [inline ios/Mattermost/Info.plist:85]:
```suggestion
	<string>Send voice messages to Mattermost</string>
```

File: ios/Mattermost/Info.plist (hunk 2 above)
Comment (cwarnermm) [reply ios/Mattermost/Info.plist:69]: Question: Is there a specific reason why ``Photo`` and ``Video`` are capitalized here? If not, let's lowercase them, please.

File: ios/Mattermost/Info.plist (hunk 7 above)
Comment (cwarnermm) [reply ios/Mattermost/Info.plist:81]: Question: Is there a specific reason why ``Photo`` and ``Video`` are capitalized here? If not, let's lowercase them, please.

File: ios/Mattermost/Info.plist (hunk 3 above)
Comment (avinashlng1080) [reply ios/Mattermost/Info.plist:73]: This happens within the Mattermost app itself.  No external app is involved.  The code would request the permission to the user to allow it to get its location.  For example, it allows the MM app to know which country the user is in.

File: ios/Mattermost/Info.plist (hunk 4 above)
Comment (avinashlng1080) [reply ios/Mattermost/Info.plist:75]: you have 2 modes for the location permission.  One is allowing the MM app to know the user's location only when the user is using the app and the other is always granted irrespective if the app is in use.  This is as per iOS protocol.

File: ios/Mattermost/Info.plist (hunk 6 above)
Comment (avinashlng1080) [reply ios/Mattermost/Info.plist:79]: I'm not sure if we actually need this permission. I removed it and the app built properly.  @enahum  please let me know if I can remove it.

File: ios/Mattermost/Info.plist (hunk 3 above)
Comment (cwarnermm) [reply ios/Mattermost/Info.plist]: Thanks, @avinashlng1080! Given the context you provided, I'm leaning towards "share" rather than "send".

**Review by cwarnermm — APPROVED** [review]
Thank you!

File: ios/Mattermost/Info.plist [hunk 10]
```diff
@@ -9,7 +9,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>Mattermost Beta</string>
+	<string>$(PRODUCT_NAME) Beta</string>
```
Comment (enahum) [inline ios/Mattermost/Info.plist:12]: no need to change this line, please revert

```suggestion
	<string>Mattermost Beta</string>
```

**Review by enahum — CHANGES_REQUESTED** [review]

**Review by enahum — APPROVED** [review]

**Pull request merged**